OpenRA.Game/Graphics/SpriteFont.cs

```diff
@@ -239,6 +271,128 @@ GlyphInfo CreateGlyph(Pair<char, Color> c)
 			return g;
 		}
 
+		float[] CreateCircularWeightMap(int r)
+		{
+			// Create circular weight maps that are used by CreateContrastGlyph for
+			// both the structuring element and to weight the resulting pixel value.
+			// The output is a 2 * r + 1 square array giving the pixel intersection
+			// with a circle of radius (r + 0.5).
+			//
+			// Example output for r=1:
```

Comment:
In case anybody wants to regenerate these:
```csharp
Console.WriteLine("Example output for r={0}:", r);
for (var j = 0; j <= 2 * r; j++)
	Console.WriteLine(elem.Skip(j * stride).Take(stride).Select(x => x.ToString("F2")).JoinWith(" "));

Console.WriteLine();
```